README.md

```diff
@@ -11,33 +11,17 @@ This repository contains a library of tools for implementing and consuming [OPTi
 
 ## Status
 
-Both the OPTiMaDe specification and this repository are **under development**.
+Both the OPTiMaDe specification and this repository are **under development**. 
 
-## Installation (Index Meta-Database)
+The latest stable version can be obtained from [PyPI](https://pypi.org/project/optimade) `pip install optimade` or by cloning this repository `git clone git@github.com:Materials-Consortia/optimade-python-tools latest`.
```

Comment:
I don't think there is a need to create the `latest` tag - it will naturally be `master`, which is also the default branch.

Reply:
If that's how we're intending to use `master` then that's fine, though major releases will invovle some large PRs... or just using a different branch of course.

Reply:
Yeah. I think we should use the recommendation that users pin to a specific version, since `master` will _always_ be the latest version, and therefore may also introduce backwards-incompatible changes. Right? Or simply use the same dependency scheme we have employed, i.e. `optimade~=0.3.3` to keep to the `0.3.x` versions, or `optimade~=0.3` to keep to the `0.x` versions. This will of course mean we should keep publishing to PyPI, but I don't see a problem with that?